ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.helpers.js

```diff
@@ -0,0 +1,26 @@
+import { visit } from '../../../helpers/visit';
+
+const basePath = '/main/vulnerabilities/workload-cves/';
+
+export function visitWorkloadCveOverview() {
+    visit(basePath);
```

Comment:
What do you think about an intercept for the primary request of the page?

```js
visit(basePath, getRouteMatcherMapForGraphQL(['getImageCVEList']));
```

Reply:
It didn't seem necessary in this case, since the tests were asserting on elements that would only exist if the request had completed successfully. Adding an intercept here seemed like relying on an implementation detail since there were alternatives that could be used in a semantic way.